.circleci/config.yml

```diff
@@ -76,6 +76,21 @@ jobs:
             docker push janx/firefox-hg
           no_output_timeout: 30m
 
+  rust:
```

Comment:
Nit: This will successfully create a `rust` Circle CI job, but it won't actually be run unless you include it in the Circle CI "workflow" at the bottom of this file.

Reply:
Also, it would be nice to tweak the "workflow" below to have a `build` step on any branch, and only `deploy` from the `master` branch (as mentioned on IRC, it currently only `build & deploy`s from the `master` branch, which prevents having CI on pull requests like this one).